### ISSUE TITLE
Make the optimizers' parameters public

### ISSUE THREAD

**Issue by rychuelektryk** [Kotlin/kotlindl#536]
Hi,

This is not a big issue, but still I don't see point in making configuration parameters private in optimizers. I would like to log for example what learningrate was set on a specific instance of a optimizer and right now it cannot be done.

**Comment by zaleslaw** [comment]
Probably, no reason to keep it private

**Comment by zaleslaw** [comment]
@rychuelektryk what lind of configuration parameters do you mean, something like this?
[link to learning rate](https://github.com/Kotlin/kotlindl/blob/master/tensorflow/src/main/kotlin/org/jetbrains/kotlinx/dl/api/core/optimizer/SGD.kt#L22)
